generatedocs/data/actions/clickactionbutton/description.md

```diff
@@ -0,0 +1,4 @@
+## ClickActionButton action
+
+Simulate clicking an action-button
```

Comment:
Maybe change to:
"Simulate clicking an action-button."
By the way, do we need an explanation of what an "action-button" is?

Reply:
Yes, I think having a brief explanation here of the supported actions of an action button might be useful.